testdata/p4_16_samples_outputs/psa-example-dpdk-externs.p4.bfrt.json

```diff
@@ -55,217 +55,6 @@
       "data" : [],
       "supported_operations" : [],
       "attributes" : ["EntryScope"]
```

Comment:
We will have to add support for DpdkMeter and DpdkCounter in bfRT json and P4info.